packages/backend/src/database/entities/userAttributes.ts

```diff
@@ -16,6 +16,15 @@ export type DbOrganizationMemberUserAttribute = {
     value: string;
 };
 
+export type DbGroupUserAttribute = {
+    group_id: number;
```

Comment:
The migration doesn't mention a column `group_id`

Reply:
Updated.